Content.Server/DeviceNetwork/Components/NetworkConfiguratorComponent.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Content.Server.DeviceNetwork.Systems;
+using Content.Shared.Sound;
+
+namespace Content.Server.DeviceNetwork.Components;
+
+[RegisterComponent]
+[Friend(typeof(NetworkConfiguratorSystem))]
+public sealed class NetworkConfiguratorComponent : Component
+{
+    /// <summary>
+    /// The list of devices stored in the configurator-
+    /// </summary>
+    [DataField("devices")]
+    public Dictionary<string, EntityUid> Devices = new();
+
+    /// <summary>
+    /// The entity containing a <see cref="DeviceListComponent"/> this configurator is currently interacting with
+    /// </summary>
+    [DataField("activeDeviceList")]
+    public EntityUid? ActiveDeviceList = null;
```

Comment:
I don't think this needs to be a data-field? Unless the goal is to support saving currently open UIs, but that would require more work.

Reply:
Afaik every component field should be a data field so we can eventually get full server states (That's see getting mentioned sometimes). I can remove it if you want.

Reply:
Yeah in general it should, but this specifically involves the BUI stuff, and I have no idea how or even IF BUIs will be saved.